addons/web/static/src/scss/popover.scss

```diff
@@ -56,7 +56,7 @@
 
     position: fixed;
     top: 0; // will be updated in js
-    left: 0; // but let's set it in the viewport in case of test/debug
+    left: 0#{"/*rtl:ignore*/"}; // but let's set it in the viewport in case of test/debug
```

Comment:
Hello, is this a magic instruction? I didn't know about that.
But shouldn't we just remove the whole line if it is making an issue and there is a comment on it saying it is useless?
Same for top ?

Reply:
Actually I can't reproduce the issue, even without the fix.
How could I reproduce it?

Reply:
@seb-odoo I removed left: 0 and top: 0 lines from css and tested with other popovers as well in LTR and RTL mode and it works well without those lines so it seems useless.

>> is this a magic instruction
Regarding your question, yes, instruction ```#{"/*rtl:ignore*/"}``` is given for rtlcss preprocessor to ignore that line while processing and converting css to RTL specific.

Reply:
Thanks, it should be good now.